romeo/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+pub mod config;
```

Comment:
You have done a pretty good job of commenting, but could you add #![forbid(missing_docs)]? Good time to add is now :P

Reply:
Agreed, will do